styles.css

```diff
@@ -0,0 +1,335 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+  font-family: 'Karla', sans-serif;
+}
+
+h1 {
+  color: #2B547E;
+}
+
+h2 {
+  color: #2B547E;
+
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 50px;
+  width: 20%;
+
+}
+
+.header-container p {
+  color: #81B741;
+  font-size: 1.2rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+
+
+.cart-container {
+  display: flex;
+
+}
+
+
+.cart-icon {
+  display: flex;
+  width: 2rem;
+  position: absolute;
```

Comment:
the positioning of this isn't quite right. It is hard to explain over email/comments. Come see me and I'll let you know what I see.